samples/nrf9160/mqtt_simple/README.rst

```diff
@@ -0,0 +1,79 @@
+.. _mqtt_simple_sample:
+
+nRF9160: Simple MQTT
+####################
+
+The Simple MQTT sample demonstrates how to easily connect an nRF9160 SiP to an MQTT broker and send and receive data.
+
+Overview
+*********
+
+The sample connects to an MQTT broker and publishes whatever data it receives on the configured subscribe topic to the configured publish topic.
+
+Requirements
+************
+
+* The following development board:
+
+  * nRF9160 DK board (PCA10090)
+
+* :ref:`secure_boot` must be programmed on the board.
+
+  The sample is configured to compile and run as a non-secure application on nRF91's Cortex-M33.
+  Therefore, it requires the :ref:`secure_boot` bootloader that prepares the required peripherals to be available for the application.
+
+Building and running
+********************
+
+This sample can be found under :file:`samples/nrf9160/mqtt_simple` in the |NCS| folder structure.
+
+The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
```

Comment:
This line here was reworked in the other LTE samples, please have a look at the `asset_tracker` instructions for building. In particular, we should specify that the sample should be built against nrf9160_pca10090_ns_.

Reply:
@joakimtoe Please address this comment. We should specify which flavour of the `nrf9160_pca10090` each sample should be built for. This one builds agains `nrf9160_pca10090ns`.

Reply:
```suggestion
The sample is built as a non-secure firmware image, thus with ``CONFIG_TRUSTED_EXECUTION_NONSECURE=y`` set in ``prj.conf``.
```
Done. But not sure I agree, beause this sample can be built for any nrf9160 board. This sample is not depending on a specific board as I see it, only nrf9160.